nxpprog.py

```diff
@@ -1191,8 +1162,10 @@ def main(argv=None):
             log(" %s" % val)
         sys.exit(0)
 
-    if len(vars(args)) == 0:
+    print(len(vars(args)))
+    if not len(sys.argv) > 1:
         parser.print_help()
+        sys.exit(0)
```

Comment:
Is this necessary? I think that argparse will print a small help text if no required arguments are passed.